src/enftun.c

```diff
@@ -148,10 +160,17 @@ enftun_tunnel(struct enftun_context* ctx)
     if (rc < 0)
         goto free_ndp;
 
+    rc = enftun_conn_state_init(&ctx->conn_state);
```

Comment:
Do this init in the `enftun_context_init` function.  All the required parameters (that is, none) are available at that point.

Reply:
As written here, `enftun_conn_state_init` is called after `enftun_conn_state_prepare`, which is backwards.  The init function is a no-op right now, so it doesn't hurt anything, but it's technically wrong.